scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -137,46 +137,42 @@ object FileSource {
   }
 
   /**
-   * @return whether globPath contains a _SUCCESS file
+   * Returns true if, for every file matched by globPath, there is a _SUCCESS file present
+   * in its parent directory.
    */
-  def globHasSuccessFile(globPath: String, conf: Configuration): Boolean = allGlobFilesWithSuccess(globPath, conf, hiddenFilter = false)
-
-  /**
-   * Determines whether each file in the glob has a _SUCCESS sibling file in the same directory
-   * @param globPath path to check
-   * @param conf Hadoop Configuration to create FileSystem
-   * @param hiddenFilter true, if only non-hidden files are checked
-   * @return true if the directory has files after filters are applied
-   */
-  def allGlobFilesWithSuccess(globPath: String, conf: Configuration, hiddenFilter: Boolean): Boolean = {
```

Comment:
why are you deleting the method? Generally, unless we really need to, it is nice to keep things around and not break compatibility.